### PR TITLE
doc: Add example how to enforce Kubernetes namespace boundaries

### PR DESCRIPTION
**Pull request by tgraf**
Suggested-by: Brandon Richins
Signed-off-by: Thomas Graf <thomas@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4869)

### CODE REVIEW

**Comment by tgraf** [comment]
test-docs-please

**Review by aanm — APPROVED** [review]

File: Documentation/policy/language.rst [hunk 1]
```diff
@@ -776,6 +776,35 @@ namespaces are automatically being taken into account:
 	  ``endpointSelector`` always applies to pods of the namespace which is
 	  associated with the CiliumNetworkPolicy resource itself.
 
+Example: Enforce namespace boundaries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example demonstrates how to enforce Kubernetes namespace based boundaries
```
Comment (ianvernon) [inline Documentation/policy/language.rst]: namespace-based

File: Documentation/policy/language.rst [hunk 2]
```diff
@@ -776,6 +776,35 @@ namespaces are automatically being taken into account:
...
+for the namespaces ``ns1`` and ``ns2`` by enabling default deny on all pods of
```
Comment (ianvernon) [inline Documentation/policy/language.rst]: default-deny

**Review by ianvernon — APPROVED** [review]

**Comment by tgraf** [comment]
test-docs-please

**Pull request merged**